sugarloaf/src/sugarloaf.rs

```diff
@@ -356,8 +367,17 @@ impl Sugarloaf<'_> {
                     self.graphics.clear_top_layer();
                 }
 
+                self.filters_brush.render(
+                    &self.ctx,
+                    &mut encoder,
+                    &frame.texture,
+                    &frame.texture,
+                    self.framecount,
```

Comment:
i think we can move framecount to inside of filters_brush instead of sugarloaf.rs